suites/quincy/cephfs/tier-2_fs_scale.yaml

```diff
@@ -114,7 +114,7 @@ tests:
       desc: "Snapshots for single volume"
       module: cephfs_scale.max_snapshots_single_volume.py
       name: cephfs Scale test for max Snapshots
-      polarion-id: "CEPH-83573520"
+      polarion-id: "CEPH-83575405"
```

Comment:
Could we add "cephfs Scale test for max Snapshots under single subvolume" in name? Also similar is needed in description

Reply:
Addressed